test/test_quantization.py

```diff
@@ -699,6 +698,34 @@ def test_single_layer(self):
         result_script = get_forward(model_script._c)(self.calib_data[0][0])
         self.assertEqual(result_eager, result_script)
 
+    @_tmp_donotuse_dont_inline_everything
+    @unittest.skip("This doesn't work right now, re-enable after fold_convbn is fixed")
+    def test_conv_bn(self):
+        r"""Compare the result of quantizing conv + bn layer in
+        eager mode and graph mode
+        """
+        # eager mode
+        conv_model = AnnotatedConvBnModel().eval()
+        conv_model_to_script = ConvBnModel().eval()
+        # copy the weight from eager mode so that we can
+        # compare the result of the two quantized models later
+        conv_model_to_script.conv.weight = torch.nn.Parameter(conv_model.conv.weight.detach())
+        fuse_modules(conv_model, ['conv', 'bn'], inplace=True)
+        model_eager = quantize(conv_model, default_eval_fn,
+                               self.img_data)
+        qconfig_dict = {
+            '': default_qconfig
+        }
+        model_script = quantize_script(
```

Comment:
We need to somehow verify that we're actually running quantized ops. If for some reason fusion doesn't work, this test won't catch it.

Reply:
do you have similar tests for graph executor?